core/src/main/java/org/mskcc/cbio/oncokb/util/TumorTypeUtils.java

```diff
@@ -31,14 +31,19 @@
         addAll(getAllSpecialTumorOncoTreeTypes());
     }};
 
-    private static List<TumorType> allOncoTreeSubtypes =
-        getOncoTreeSubtypesByCancerTypesFromSource(allOncoTreeCancerTypes);
+    private static List<TumorType> allOncoTreeSubtypes = getOncoTreeSubtypesFromSource();
     private static Map<String, TumorType> allNestedOncoTreeSubtypes = getAllNestedOncoTreeSubtypesFromSource();
     private static Map<String, List<TumorType>> questTumorTypeMap = null;
     private static Map<String, List<TumorType>> cbioTumorTypeMap = null;
+    private static final ImmutableList<String> LiquidTumorTissues = ImmutableList.of(
+        "Lymph", "Blood", "Lymphoid", "Myeloid"
```

Comment:
So the tissue name changed from BLOOD to Blood?

Reply:
The name is always Blood. BLOOD is the code, but we don't use it on the tissue level.